sway/ipc-json.c

```diff
@@ -602,6 +602,18 @@ static void ipc_json_describe_view(struct sway_container *c, json_object *object
 	json_object_object_add(object, "inhibit_idle",
 		json_object_new_boolean(view_inhibit_idle(c->view)));
 
+	const char *sandbox_engine = view_get_sandbox_engine(c->view);
+	json_object_object_add(object, "sandbox_engine",
```

Comment:
Seems like we should be adding these new props to the IPC man page as well.

Reply:
Already added to `sway-ipc.7.scd`

Reply:
Oh, my bad, I missed it somehow.